demofile-parsing/Round.js

```diff
@@ -11,6 +11,7 @@ class Round {
     this.highRateTimes = [];
     this.highlights = [];
     this.playerKills = {};
+    this.firstKills = {};
```

Comment:
Whats the purpose for the this hash?

Reply:
@tjlam Yah I'm a bit dehydrated....since we're always writing over the {killerName, killCount} hash .....so I thought I might be a bit 'cleaner' to have a {killerName, firstKillTime} hash versus a triple key object { killerName, killCount, firstKillTime} and always putting firstKillTime in auxiliary storage as we update the hash

Reply:
Ok makes sense kinda. Not sure what we use it for though? I think if a player gets a 4k or 5k, we should override the `this.highlights = [firstKilltime, firstkillTime+30]`

Reply:
@tjlam done my good sir <3